### DISCUSSION TITLE
How could I implement default we=true for all hw writeable fields.

### DISCUSSION THREAD

**Discussion by fevredream** [SystemRDL/systemrdl-compiler#215] [General]
For overall ppa, I would like to add we property for all hw writeable fields.
I could add we property for all fields, or regs if all sub-fields are hw writeable, individually.
Here I would like to ask if there is something l could add like a systemrdl property or a compiler-specific option for this, with less effort.

Add "default we = true;" to addrmap may result in an error for trying to assign we property to non-hw-writeable fields.

**Comment by amykyta3** [comment]
I am not aware of a SystemRDL language mechanism that would do this, sorry.
